src/components/utils/error/ErrorHandler.js

```diff
@@ -54,9 +55,15 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 function ClientInfo(props) {
+    const [browser, setBrowser] = useState();
```

Comment:
Since this is now in a page, access to `window` is only available from `useEffect`